src/dmclock_recs.h

```diff
@@ -33,20 +33,20 @@ namespace crimson {
 
     struct ReqParams {
       // count of all replies since last request; MUSTN'T BE 0
-      uint32_t delta;
+      uint64_t delta;
```

Comment:
i am just curious why we want to use `uint64_t` for delta and rho now? because we could be running into integer overflow with `uint32_t`. as IMO,  `uint32_t` is able to represent a fairly large number already.

Reply:
This is a very good point. I think we had a mention that rho and delta would be u64s on our etherpad when we discussed it. But that seems to large. 4 billion should be more than enough to count the number of replies received from other servers. I'll switch it back.

Reply:
i can imagine a strategy that sets cost=bytes in a request, and radosgw will happily serve requests bigger than 4G. but given a Cost=uint32_t, you could easily adapt the strategy to some fraction of the bytes instead

are we doing much math with these values, where there'd be risk of an intermediate multiplication overflowing?

it seems like the benefits of uint32_t would be less overhead in osd messages, and less per-request memory for the dmclock prio queue? those saving could definitely add up

Reply:
I can imagine someone trying that strategy as well, although I'd hope people would think in terms of blocks rather than bytes. Assuming 4K blocks a request of 4G would result in a cost of 1M. You'd then need a request of 16T to overflow cost.

So I think we're piecing together a case for uint32_t for cost. @tchaikov -- your thoughts?